types/graphql/index.d.ts

```diff
@@ -6,6 +6,7 @@
 //                 Firede <https://github.com/firede>
```

Comment:
Please update the version number for graphql. I think these apply to 0.11?

Reply:
@minestarks Done ✅